net/metrics.go

```diff
@@ -22,7 +23,9 @@ func (h incomingConnectionsStatsHandler) HandleRPC(ctx context.Context, rpcStats
 }
 
 func (h incomingConnectionsStatsHandler) TagConn(ctx context.Context, tagInfo *stats.ConnTagInfo) context.Context {
-	metrics.IncomingConnectionTimestamp.WithLabelValues(tagInfo.RemoteAddr.String()).SetToCurrentTime()
+	remoteAddr := tagInfo.RemoteAddr.String()
```

Comment:
use https://pkg.go.dev/net#SplitHostPort

Reply:
Thanks for the heads-up. Anyway, I've removed the whole thing. We'll just track outgoing connections.